docs/sources/open-source.md

```diff
@@ -133,38 +155,56 @@ lt --port 8080 -s pretty-turkey-83 --print-requests
     socket_mode_enabled: false
   ```
 
-6. Go to your "OnCall" -> "Env Variables" and set:
+6. Set environment variables by navigating to your Grafana OnCall, then click **Env Variables** and set the following:
```

Comment:
```suggestion
1. Set environment variables by navigating to your Grafana OnCall, then click **Env Variables** and set the following:
```
Where is this? In ChatOps -> slack? It's not clear and these fields aren't available in Cloud OnCall